lib/shortcodes/property-overview.php

```diff
@@ -64,6 +64,12 @@ public function __construct()
               'type' => 'text',
               'default' => ''
             ),
+            'wpp_agents' => array(
+              'name' => sprintf(__('Agent %s', ud_get_wp_property('domain')), \WPP_F::property_label()),
+              'description' => __('If not empty, result will show property, which Agent ID is set.', ud_get_wp_property()->domain),
```

Comment:
`sprintf( __('Optional. If set, shows only %s to which Agent belongs', ud_get_wp_property()->domain), \WPP_F::property_label( 'plural' ) )`